modules/ROOT/pages/message-type-receive-from-partners.adoc

```diff
@@ -0,0 +1,83 @@
+= Receive from Partners Message Type Settings
+
+The Receive from Partners message type defines the way in which Anypoint Partner Manager transforms the inbound transactions sent from your partner to your organization. These transactions typically follow JSON, EDIFACT, X12, or XML standards.
+
+== EDIFACT
+
+Use the following settings to define EDIFACT as the protocol for the Receive from Partners message type:
+
+[%header%autowidth.spread]
+|===
+|Setting |Description |Default | Required
+|Usage | Message usage. Allowed value is `Receive from partners`. |  | Yes
+|Format | Format of the message. Allowed value is `EDIFACT`.| |Yes
+|Version | EDIFACT version, such as `d10b` or `d96a`. |  |Yes
+|Message type |
+EDIFACT message type, such as `ORDERS`, `DESADV`, or `INVOIC`.
+ |  | Yes
+|Reference identifiers and attributes mapping a| Uploaded DataWeave map that extracts the reference identifiers and custom attributes from the payload received from the partner:
+
+* The extracted reference identifier values are used for dynamic routing into the appropriate message flow.
+* The extracted custom attributes enable you to have visibility into the lifecycle of your B2B workflow. For more information, see xref:use-custom-attributes.adoc[Using Custom Messaging Attributes with Message Types].
+| |No
+|Schema a|EDI schema for the message type. Options include:
+
+* *Use standard EDIFACT-<version>* +
+Applies the standard EDIFACT schema for the configured format and version.
+* *Customize* +
+Enables you to import a customized xref:connectors::x12-edi/x12-edi-schema-language-reference.adoc[EDI schema language] file, if your need to make adjustments to the standard EDIFACT schema.
```

Comment:
Should "your" be "you" here:

...if your need to make adjustments to the standard EDIFACT schema.